### DISCUSSION TITLE
Concealing Icons Help

### DISCUSSION THREAD

**Discussion by MVP-Harry** [nvim-neorg/neorg#408] [Q&A]
Hi there,

When I enable concealers and customize the icons (using the configuration on [this page](https://github.com/nvim-neorg/neorg/wiki/Concealing)), nvim displays some weird icons. I think it's probably because I'm missing some icons/fonts, but I don't know which one. According to the page, I tried to follow the instruction on [this](https://github.com/ronniedroid/getnf) page to download nerdfonts, but I already have most of the fonts installed, I think I'm missing the "icons". 

Below is an illustration of the problem:
![image](https://user-images.githubusercontent.com/64049111/160309044-01b3a81a-5075-42b7-ba79-52c6e695dd39.png)

Thanks in advance!

**Comment by danymat** [comment]
Hello, what happens if you remove your custom configuration and enable concealing ?
